types/qs/index.d.ts

```diff
@@ -55,6 +55,16 @@ declare namespace QueryString {
         interpretNumericEntities?: boolean;
     }
 
+    interface ParsedQs { [key: string]: string | string[] | ParsedQs | ParsedQs[] }
+
+    // The value type here is a "poor man's `unknown`". When these types support TypeScript
+    // 3.0+, we can replace this with `unknown`.
+    type PoorMansUnknown = {} | null | undefined;
```

Comment:
I switched from `unknown` (https://github.com/DefinitelyTyped/DefinitelyTyped/pull/43440#discussion_r399770343) to this so I didn't have to bump the versions of all types which depend on `qs`—including Express which started to depend on this very recently.